Aquality.Appium.Mobile/tests/Aquality.Appium.Mobile.Tests/Samples/Android/Web/WebTextBoxTest.cs

```diff
@@ -0,0 +1,38 @@
+﻿using Aquality.Appium.Mobile.Applications;
+using NUnit.Framework;
+using OpenQA.Selenium;
+
+namespace Aquality.Appium.Mobile.Tests.Samples.Android.Web
+{
+    public class WebTextBoxTest : UITest
+    {
+        private const string ValueToSubmit = "quality assurance";
+
+        [Test]
+        public void TextBoxInteraction()
+        {
+            AqualityServices.Application.Driver.Url = "https://wikipedia.org";
+            var txbSearch = AqualityServices.ElementFactory.GetTextBox(By.Id("searchInput"), "Search");
+            txbSearch.State.WaitForClickable();
+            txbSearch.Click();
+            // not yet implemented in dotnet Appium client: Assert.IsTrue(driver.isKeyboardShown(), "Keyboard should be shown when click successful");
```

Comment:
Isn't it better to place `//TODO` to not forget about such things?

Reply:
Fixed